Requirements.md

```diff
@@ -0,0 +1,13 @@
+## Software Requirements Specifications for Spring 2020 "Intro to Software Engineering" "Craft Topography" Project at Wright State Univesity
+### Functional Requirements
+  #### User Story A
+   - Req 1.0 Craft implementation shall create a terrain from a topographic image
+   - Req 1.1 Topographic image shall be a .png image
+   - Req 1.2 Pixel Values shall be normalized to represent a valid range
+   - Req 1.3 Valid range shall be a minimum of 20 blocks and a maximum of 100 blocks
+   - Req 3.0 One pixel in topographic image shall correspond to a 2x2 block area
+  #### User Story B
+   - Req 2.0 Block types shall be more varied
+   - Req 2.1 Blocks over 80 blocks high shall be snow blocks
+   - Req 4.0 The minimum block height shall be 20 blocks
+   - Req 4.1 The maximum block height shall be 100 blocks
```

Comment:
Can clouds be higher than 100 blocks? Do you mean "Generated terrain blocks shall have a maximum height of 100 blocks"?